auto_submit/bin/server.dart

```diff
@@ -9,25 +9,27 @@ import 'package:auto_submit/helpers.dart';
 import 'package:shelf/shelf.dart';
 import 'package:shelf_router/shelf_router.dart';
 
-class AutosubmitServer {
-  AutosubmitServer();
+Future<Response> webhookHandler(Request request) async {
+  final String? reqHeader = jsonEncode(request.headers);
+  print('Header: $reqHeader');
```

Comment:
Import `package:logging` and use logs instead

https://dart-lang.github.io/linter/lints/avoid_print.html